### ISSUE TITLE
[Feature request] Show the battery state as percentage

### ISSUE THREAD

**Issue by marmistrz** [linuxmint/cinnamon-screensaver#213]
It would be great if the screensaver could be configured to should the battery percentage in text form, not as an icon. 

The icon display tells me much less than a sheer number.

**Comment by callagg2** [comment]
Yes, I agree, on the top right of the lock screen/(Screensaver) we could have the battery indicator - one similar in design to the Spotify volume indicator on the left. 

We could also have the clock appear here in the top right, not sure there is anything to be gained from having it floating around the screen - even someone wants it as a screensaver fine, but not sure the clock should be moving to different positions on the lockscreen.

The top right could also show any new notifications beside the clock - both alerts as and when they come in (which can be set to show or hide contents like on phones), and an ongoing notification count icon. 

For Spotify, on the lockscreen there is no progress indicator. 
We could place the the rewind, pause and wind buttons under the existing volume indicator would should add to the height of the box, this would then allow us to put a progress indicator under the album title.
There is nothing to be gained from having the album art for Spotify floating around the screen, it should just appear to the left of the volume indicator. 

Such changes then frees up the middle of screen for a a proper screensaver featuring maybe a movement of the LM icon.

**Comment by AdamRGrey** [comment]
workaround: https://forums.linuxmint.com/viewtopic.php?t=412400
